src/aws-cpp-sdk-core/source/http/curl/CurlHttpClient.cpp

```diff
@@ -751,6 +751,14 @@ std::shared_ptr<HttpResponse> CurlHttpClient::MakeRequest(const std::shared_ptr<
             ss << m_proxyScheme << "://" << m_proxyHost;
             curl_easy_setopt(connectionHandle, CURLOPT_PROXY, ss.str().c_str());
             curl_easy_setopt(connectionHandle, CURLOPT_PROXYPORT, (long) m_proxyPort);
+            if(!m_proxyCaPath.empty())
+            {
+                curl_easy_setopt(connectionHandle, CURLOPT_PROXY_CAPATH, m_proxyCaPath.c_str());
```

Comment:
looks like `CURLOPT_PROXY_CAPATH` and `CURLOPT_PROXY` were added in [7.52](https://curl.se/libcurl/c/CURLOPT_PROXY_CAPATH.html). we should likely ifdef guard this like we do the [ssl version](https://github.com/aws/aws-sdk-cpp/blob/main/src/aws-cpp-sdk-core/source/http/curl/CurlHttpClient.cpp#L721). We have users that use versions [as low as 7.29](https://github.com/aws/aws-sdk-cpp/pull/2775)